projects/aca-content/src/lib/components/details/details.component.spec.ts

```diff
@@ -128,4 +131,11 @@ describe('DetailsComponent', () => {
     fixture.detectChanges();
     expect(store.dispatch).toHaveBeenCalledWith(new SetSelectedNodesAction([node]));
   });
+
+  it('should call getInfoDrawerIcon with the provided node', () => {
+    fixture.detectChanges();
+    spyOn(nodeActionsService, 'getInfoDrawerIcon');
+    component.getInfoDrawerIcon(node.entry);
+    expect(nodeActionsService.getInfoDrawerIcon).toHaveBeenCalledWith(node.entry);
```

Comment:
What is this Unit test testing?

Reply:
I updated the unit test for checking the icon in return